armory/eval/evaluator.py

```diff
@@ -24,6 +25,42 @@
 logger = logging.getLogger(__name__)
 
 
+def tmp_output_subdir(retries=10):
```

Comment:
I would prefer this be in in some docker.utils module as there's a lot to unpack in this function and I worry it distracts from the Evaluator class which will be often looked at.

Reply:
Makes sense.

Reply:
Moved to `armory.docker.volume_utils`